FluentTerminal.App/Dialogs/CreateKeyBindingDialog.xaml.cs

```diff
@@ -47,11 +50,19 @@ public bool Meta
         public int Key
         {
             get => _key;
-            set => Set(ref _key, value);
+            set
+            {
+                // We're ignoring invalid keys.
+                if (_acceleratorKeyValidator.Valid(value))
```

Comment:
No. Currently such keybindings are simply ignored (not shown in the app menu). But in the future such keys could not be saved due to this exact line.

Reply:
I don't think we should do this. This will lead to a ton of "why am I not abel to use $key" issues. Better solution would be to use https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyoutitem.keyboardacceleratortextoverride if available or just leave it blank for older windows versions

Reply:
> This will lead to a ton of "why am I not abel to use $key" issues.

I don't think that you'll get any such complaint because these aren't usual keys for shortcuts anyway. If a user tries to enter comma symbol for example, and see that it's ignored, he'll realize immediately that it can't be used. (Try that in my branch.) Also keep in mind that current solution is also limited (`ExtendedVirtualKey` has only few more keys than `VirtualKey`).

Suggested text override is possible, but keep in mind that key accelerators aren't only about displaying shortcut text - they also trigger the menu item, so which `Key` value we should use? `0` (which is `VirtualKey.None`)? Wouldn't that trigger the menu item on `Ctrl+Shift` only, no matter our text override writes `Ctrl+Shift+,`? What I'm trying to say is that it would require developing some smart logic. Is it worthwhile?

Finally, as you can see this PR includes 20+ files, 1,000+ lines, and works perfectly in vast majority of cases (for all usual shortcuts). Can you simply merge it, and consider this fix later? Is it better to make 99.9% of the users happy with the cool new feature, or keep everything back because 0.01% may complain about not being able to use comma in keyboard shortcuts?

Reply:
To put it simply: `KeboardAccelerator.Key` property is of type `VirtualKey`, so it can accept only `VirtualKey`. I haven't introduced this limitation - Microsoft did. And if you try to set it to something like `(VirtualKey) (int) ExtendedVirtualKey.Comma`, the app crashes in very ugly way - it took me hours to realize what's happening, and it turned out to be our `Ctrl+Shift+,` shortcut.

Reply:
I've spent few more hours trying to find a solution which will satisfy you, but no luck. I've managed to show comma shortcuts in the menu:
![image](https://user-images.githubusercontent.com/6330857/69595950-46f9a900-1001-11ea-9f42-8400a1a9e31d.png)

But it doesn't work. It cannot work as a regular keyboard accelerator (due to an invalid key). On the other side, I cannot catch keyboard events to handle them manually (if the menu is open, neither main window nor Hamburger button are receiving keyboard events).

You can see that in my branch: https://github.com/jumptrading/FluentTerminal/tree/ap/menuaccelerators3